components/TestResult/vue/TestResultVue.cy.tsx

```diff
@@ -0,0 +1,63 @@
+/// <reference types="cypress" />
+import { mount } from 'cypress/vue'
+import assertions from '../assertions'
+import TestResult from './TestResult.vue'
+import {
+  TestResultData,
+  TestResults,
+} from '@cypress-design/constants-testresult'
+import Button from '@cypress-design/vue-button'
+import { IconActionTestReplay } from '@cypress-design/vue-icon'
+import { IconChevronRightSmall } from '@cypress-design/vue-icon'
+import { ref } from 'vue'
+
+const TestResultSut = (result: TestResultData) => {
+  const showGroupBox = ref(false)
+  return (
+    <TestResult
+      status={result.status}
+      names={result.names}
+      flaky={result.flaky}
+      modified={result.modified}
+      added={result.added}
+    >
+      <Button
+        variant="outline-light"
+        size="32"
+        class="!px-[8px] @lg/test-result:!px-[12px] h-[32px]"
+      >
+        <IconActionTestReplay />
+        <span class="hidden @lg/test-result:inline ml-[8px]">Test Replay</span>
+      </Button>
+      <Button
+        variant="outline-light"
+        size="32"
+        class="!px-[8px] hidden @xl/test-result:inline-block h-[32px]"
+        onClick={() => (showGroupBox.value = !showGroupBox.value)}
+      >
+        <IconChevronRightSmall
+          stroke-color="gray-500"
+          class={{
+            'transition-transform transform': true,
+            'rotate-90': showGroupBox,
+          }}
+        />
+      </Button>
+    </TestResult>
+  )
+}
+
+describe('<TestResult/>', () => {
+  function mountStory() {
+    mount(() => (
+      <div class="p-[16px] antialiased">
```

Comment:
@elevatebart Any chance there's a better way to handle the `antialiased` class in a more global way? It would be nice that we could make sure that we have some base styles or resets that can apply for every property using `cypress-design`. Otherwise, we would have to repeat some classes all throughout our DS components in order for things to render as expected downstream.

Reply:
Done. Please check before I merge ;)

Reply:
@elevatebart Thanks. Will this automatically kick in then in the component docs and component tests as well? This came up yesterday when Emil thought I was applying a medium font weight to the test names, but when I dug in, it was clear that this wasn't the case. In the end, it came down to the fact that we didn't have this anti-aliasing property applied across the board.